src/WarcraftLegacies.Source/Quests/Skywall/QuestThroneWind.cs

```diff
@@ -21,16 +21,14 @@ public sealed class QuestThroneWind : QuestData
     /// </summary>
     /// <param name="rescueRect">Units in this area will start invulnerable and be rescued when the quest is complete.</param>
     public QuestThroneWind(Rectangle rescueRect) : base("The Throne of the Four Winds",
-      "We have lost contact with the Elemental Realm, conquer Uldum to access the Vortex Pinnacle.",
+      "We still don't have full control of the Elemental Realm. If we defeat the trolls of Zul'Farrak, we could use their power to secure dominion over the Throne of the Four Winds.",
```

Comment:
```suggestion
      "We still don't have full control of Skywall. If we defeat the trolls of Zul'Farrak, we could use their power to secure dominion over the Throne of the Four Winds.",
```
Because I don't think this quest is about conquering the _entire_ plane.